src/analyses/uninit.ml

```diff
@@ -47,6 +47,7 @@ struct
     | _ ->
       M.warn "Access to unknown address could be global"; []
 
+  (* TODO: unused? remove? *)
```

Comment:
Is it not used via `is_expr_initd` -> `vars` -> `varoffs` -> `access_one_byval`?

Reply:
You're right. I accidentally added the TODO to the wrong function, the similarly named `access_byval` below is actually unused.